### DISCUSSION TITLE
Could not upgrade or reinstall new updates

### DISCUSSION THREAD

**Discussion by tareksahalia** [Chia-Network/chia-blockchain#3113] [Q&A]
Hello,
I installed chia 1.1.1 in my ubuntu 20.x.x version. now I am not able to upgrade to the latest version.
when upgrade
- when executing **chia stop -d all** I get **Command 'chia' not found**,
when I tried to re-clone and install again
- when I execute  **. ./activate** I get **bash: ./activate: No such file or directory**

Now I do have the 1.1.1 chia version, and it still showing me that it is farming. I don't know if this is true cause Chia official announcement says that the earlier versions will stop working.
Any help is much appreciated !!
thanks

**Comment by KennySoh** [comment]
I am hitting the same issue

**Reply by KennySoh** [reply]
This helped me hope it helps u too
https://itsfoss.com/repository-does-not-have-release-file-error-ubuntu/

Based on the my error, it had something to do with ubuntu
```
apt-get is /usr/bin/apt-get
Installing on Ubuntu/Debian 20.04 LTS
Hit:1 http://security.ubuntu.com/ubuntu focal-security InRelease    
Hit:2 http://sg.archive.ubuntu.com/ubuntu focal InRelease           
Hit:3 http://ppa.launchpad.net/atareao/atareao/ubuntu focal InRelease
Hit:4 http://sg.archive.ubuntu.com/ubuntu focal-updates InRelease        
Ign:5 http://ppa.launchpad.net/cpug-devs/ppa/ubuntu focal InRelease      
Hit:6 http://sg.archive.ubuntu.com/ubuntu focal-backports InRelease
Err:7 http://ppa.launchpad.net/cpug-devs/ppa/ubuntu focal Release
  404  Not Found [IP: 91.189.95.85 80]
Reading package lists... Done                      
E: The repository 'http://ppa.launchpad.net/cpug-devs/ppa/ubuntu focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

```

**Reply by fjbotto** [reply]
I faced this issue on Ubuntu 20, and I disabled the PPA from software & updates then re-run the install. [Here some more detailed steps](https://www.omgubuntu.co.uk/2019/12/how-to-remove-ppa-ubuntu)
